doc/main/tbb_userguide/attach_flow_graph_to_arena.rst

```diff
@@ -0,0 +1,62 @@
+.. _attach_flow_graph_to_arena:
+
+Attach Flow Graph to arbitrary Task Arena
+======================
+
+
+During construction a ``graph`` object captures a reference to the arena of the thread that
+constructed the object. Whenever a task is spawned to execute work in the graph, the tasks
+are spawned in this arena, not in the arena of the that caused the task to be spawned.
+All tasks are spawned into a single arena, the arena of the thread that constructed the graph
+object.
+
+
+::
+
+
+       task_arena a{2};
```

Comment:
what should this example demonstrate to users?

Reply:
Adding tittles to both examples